ckanext/gdi_userportal/plugin.py

```diff
@@ -12,16 +12,31 @@
     get_publisher_list,
     get_theme_list,
     scheming_package_show,
-    get_with_url_labels
+    get_with_url_labels,
 )
 from ckanext.gdi_userportal.logic.auth.get import config_option_show
+import json
 
 
 class GdiUserPortalPlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IConfigurer)
     plugins.implements(plugins.IFacets, inherit=True)
     plugins.implements(plugins.IAuthFunctions)
     plugins.implements(plugins.IActions)
+    plugins.implements(plugins.IPackageController)
+
+    _dcatap_fields_to_normalize = [
```

Comment:
Is it possible to make it field name agnostic (requesting schema with toolkit.get_action('scheming_dataset_schema_show') and selecting fields with preset = multiple_text so fields are not hard-coded?

Reply:
I would recommend we keep it like that. I can open another ticket to invastigate it appropriately, what do you think?